docs/content/preview/develop/learn/ttl-data-expiration-ycql.md

```diff
@@ -130,39 +138,40 @@ Note that there is some tradeoff here between the number of files created and re
+In some fresh dataset cases, new data will be backfilled into the database before the application is turned on. This backfilled data may have a value-level TTL associated with it that is significantly lower than the `default_time_to_live` property on the table, with the desired effect being that this data be removed earlier than the table TTL would allow. By default, such data does not expire early. However, the `file_expiration_value_ttl_overrides_table_ttl` flag can be used to ignore table TTL and expire solely based on value TTL.
 
 {{< warning title="Warning" >}}
-When using the `file_expiration_value_ttl_overrides_table_ttl` flag, be sure to set the flag back to `false` *before* all data with value-level TTL (for example, backfilled data) has fully expired. Failing to do so can result in unexpected loss of data. For example, if the `default_time_to_live` is 90 days, and data has been backfilled with value-level TTL from 1 day to 89 days, it is important that the `file_expiration_value_ttl_overrides_table_ttl` flag be set back to `false` within 89 days of data ingestion to avoid data loss.
+When using the `file_expiration_value_ttl_overrides_table_ttl` flag, be sure to set the flag back to `false` before all data with value-level TTL (for example, backfilled data) has fully expired. Failing to do so can result in unexpected loss of data. For example, if the `default_time_to_live` is 90 days, and data has been backfilled with value-level TTL from 1 day to 89 days, it is important that the `file_expiration_value_ttl_overrides_table_ttl` flag be set back to `false` within 89 days of data ingestion to avoid data loss.
 {{< /warning >}}
 
 ### Configuring for existing YCQL datasets
 
-To convert existing YCQL tables to ones configured for file expiration, the same TServer flag values as above can be used. However, expect a **temporary 2x space amplification** of the data should be expected in this case. This amplification happens because the existing file structure will have kept most data in a single large file, and that file will now be excluded from compactions going forward. Thus, this file will be unchanged until its contents has entirely expired, roughly *TTL amount of time* after the file expiration feature was configured.
+To convert existing YCQL tables to ones configured for file expiration, the same TServer flag values as above can be used. However, a temporary 2 times space amplification of the data should be expected in this case. This amplification happens because the existing file structure will have kept most data in a single large file, and that file will now be excluded from compactions going forward. Thus, this file will be unchanged until its contents has entirely expired, roughly TTL amount of time after the file expiration feature was configured.
```

Comment:
Thus, this file will be unchanged until its contents have entirely expired

Reply:
fixed